recipes/1-Insights/conditioned-insights.ipynb

```diff
@@ -0,0 +1,309 @@
+       "rmse                   NaN\n",
+       "r2                     NaN\n",
+       "Name: target, dtype: float64"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "global_stats = t.get_prediction_stats()['target']\n",
+    "global_stats"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Local Prediction Stats\n",
+    "\n",
+    "In addition to global prediction stats, Howso has the unique ability to provide local prediction stats. These prediction stats are more finely tuned to the exact cases in which you are interested in. This can be extremely valuable for workflows such as data exploration and it provides more nuanced performance metrics. The more variance there is in the performance of dataset from region to region, the more powerful this ability is."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.7"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results = t.react(\n",
+    "    test_case,\n",
+    "    context_features=context_features,\n",
+    "    action_features=action_features,\n",
+    "    details = {\n",
+    "        \"prediction_stats\": True\n",
+    "    }\n",
+    ")\n",
+    "\n",
+    "local_stats = results['details']['prediction_stats'][0]\n",
+    "local_stats['accuracy'][action_features[0]]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see how the local prediction stats differ from the global stats. Using local metrics, we can drill into specific cases. For example, in our test case which is manually set to `female`, we can get insights into how well the Trainee fits similiar cases. In the local space, this does not gurantee all of the other cases in the local space are also female, however it increases the chances that they are.\n",
+    "\n",
+    "If we see that the local accuracy drops off a lot from the global accuracy, this can demonstrate that in this region of the data, the Trainee and data do not have the same predictive performance as the rest of the Trainee and data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Conditioned Prediction Stats\n",
+    "\n",
+    "Zooming back out to global prediction stats, Howso also has the ability the condition the prediction stats by providing conditions for the context set and action set. Similar to context features and action featuers, the context set is the set queried to make predictions and the action set is the set that the predictions are made for. In other words, you use the data from the context set is the known data and the action set is the unknown data in which you are trying to predict using the context set.\n",
```

Comment:
"the ability the condition" --> "the ability to condition"

Reply:
"In other words, the data from the context set is the known information you are using to make a prediction and the action set is unknown information you are trying to predict."